controllers/deviceprofile_syncer.go

```diff
@@ -107,15 +100,15 @@ func (ds *DeviceProfileSyncer) Run(stop <-chan struct{}) {
 			// delete the device profiles on Kubernetes but not on Egdex
 			deleteKDevs := findDeleteDeviceProfile(eDevs, kDevs.Items)
 			if len(deleteKDevs) != 0 {
-				if err := deleteDeviceProfile(ds.log, ds.Client, deleteKDevs); err != nil {
-					ds.log.Error(err, "fail to delete device profiles")
+				if err := deleteDeviceProfile(ds.Client, deleteKDevs); err != nil {
+					klog.V(3).ErrorS(err, "fail to delete device profiles")
 				}
 			}
 		}
```

Comment:
Adding a log line to indicate completion of one synchronization round may be better.

Reply:
The log has been added.